src/Client/ClientBase.cpp

```diff
@@ -3068,6 +3068,8 @@ void ClientBase::init(int argc, char ** argv)
         ("max_memory_usage_in_client", po::value<std::string>(), "Set memory limit in client/local server")
 
         ("fuzzer-args", po::value<std::string>(), "Command line arguments for the LLVM's libFuzzer driver. Only relevant if the application is compiled with libFuzzer.")
+
+        ("client_log_file", po::value<std::string>(), "Path to a file for logging fatal errors in client")
```

Comment:
It can be ambiguous with the usual logs (send_logs_level) in clickhouse-local.
It's strange that the argument is named `client_log_file`, but its description is only about fatal logs.

Reply:
We don't have other logs in client, `send_logs_level` is for server logs. Currently we have only fatal logs in client (and we need them only in the CI because mane .sh tests redirect stderr), but if we add more client logs (this may be useful in some cases) we can use the same file and the same argument as for fatal logs. But I think I will rename it to `client_fatal_log_file` for now

Reply:
Ok. It will be consistent with the `server_logs_file` argument.